turbo.json

```diff
@@ -1,53 +1,25 @@
 {
   "$schema": "https://turbo.build/schema.json",
-  "globalDependencies": [
-    "tsconfig.json"
-  ],
+  "globalEnv": ["TEST_DB_URL", "CI"],
```

Comment:
aren't those only related to backend?

Reply:
yessssir, this should do the trick unless there is a better option to specify these specially for backend/test
https://github.com/l2beat/l2beat/pull/4235/commits/bc6a5ca58418635d896aa2bf64ae707c3ecfeeff